movie-mlops-project/serving/main.py

```
@@ -0,0 +1,311 @@
+            "timestamp": datetime.now().isoformat(),
+            "data": {
+                "total_count": predictions['sample_count'],
+                "predictions": predictions['results']
+            }
+        }
+        
+    except HTTPException:
+        raise
+    except Exception as e:
+        logger.error(f"예측 결과 조회 실패: {e}")
+        raise HTTPException(status_code=500, detail=f"예측 결과 조회 실패: {str(e)}")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)